templates/default/sql/dml/0.8.0.sql.erb

```diff
@@ -0,0 +1,3 @@
+REPLACE INTO variables VALUES ("featurestore_default_quota", "<%= @featurestore_default_quota %>");
+REPLACE INTO variables VALUES ("featurestore_default_storage_format", "<%= @featurestore_default_storage_format %>");
```

Comment:
Here you are just passing a node attribute, instead of using a template variable, just put the node attribute as you do in the next line. 
It makes `hopsworks::default` recipe shorter and easier to read.

Reply:
Sure, done